static/js/compose.js

```diff
@@ -496,15 +496,8 @@ exports.get_invalid_recipient_emails = function () {
     return invalid_recipients;
 };
 
-function check_unsubscribed_stream_for_send(stream_name, autosubscribe) {
-    var stream_obj = stream_data.get_sub(stream_name);
+function check_unsubscribed_stream_for_send(stream_name) {
```

Comment:
I'm a little confused about what happens to `autosubscribe` in this commit.

Reply:
In this commit (I think you're talking about the second one) I moved some of the code from `check_unsubscribed_stream_for_send` (the function we're planning to get rid of) to `validate_stream_message_address_info`, so passing `autosubscibe` as a parameter became unnecessary. Basically, this is a preliminary step to make further commits more readable.